cfme/tests/cli/test_appliance_update.py

```diff
@@ -19,13 +19,18 @@ def pytest_generate_tests(metafunc):
 
     version = current_appliance.version
     split_ver = str(version).split(".")
-    minor_build = split_ver[2]
+    try:
+        minor_build = split_ver[2]
+    except IndexError:
+        logger.exception('Caught IndexError generating for test_appliance_update, skipping')
+        pytest.skip('Could not parse minor_build version from: {}'.format(version))
 
     for i in range(int(minor_build) - 1, -1, -1):
         versions.append("{}.{}.{}".format(split_ver[0], split_ver[1], i))
 
 
 @pytest.yield_fixture(scope="function")
+@pytest.mark.uncollectif(lambda: not appliance_is_downstream())
```

Comment:
Kind of redundant since its going to skip during generation against upstream builds, before reaching this, but just-in-case?